content/docs/start/data-pipelines.md

```diff
@@ -150,8 +151,9 @@ in this case); `dvc run` already took care of this. You only need to run
 
 By using `dvc run` multiple times, and specifying <abbr>outputs</abbr> of a
 stage as <abbr>dependencies</abbr> of another one, we can describe a sequence of
-commands that gets to a desired result. This is what we call a _data pipeline_
-or [_dependency graph_](https://en.wikipedia.org/wiki/Directed_acyclic_graph).
+commands which gets to a desired result. This is what we call a <abbr>data
+pipeline</abbr> or
```

Comment:
Same problem with tooltippin' this: unnecessary (it's already defined in the same p) and a bit noisy:

![image](https://user-images.githubusercontent.com/1477535/114668110-f5b98a80-9cc5-11eb-9637-f71974d44e38.png)

Existing tooltips were strategically placed.